README.md

```diff
@@ -46,12 +46,20 @@ Citus Cloud runs on top of AWS as a fully managed database as a service and has
 
 If you're looking to get started locally, you can follow the following steps to get up and running.
 
-* Install docker-compose: [Mac][mac_install] | [Linux][linux_install]
-* (Mac only) connect to Docker VM
-  ```bash
-  eval $(docker-machine env default)
-  ```
-
+* Install Docker Community Edition and docker-compose
```

Comment:
Minor question. Could we make this a numbered list to make it consistent with our website's installation instructions?

Please feel free to merge this version first and we could then address it in a separate pull request.